Article/Article.js

```diff
@@ -3,24 +3,31 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
+    this.expandButton = document.querySelector('.expandButton');
     
+    // Using your expandButton reference, update the text on your expandButton to say "expand"
+    this.expandButton.textContent = "expand";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener("click", () => this.expandArticle(event)); //not sure if even should be passed
```

Comment:
`event` should **not** be passed here for two reasons:
1. It is not passed as a parameter, so currently it's relying on Chrome's default assignment (making this event callback not work in other browsers).
1. You are not using `event`(or any parameters) in `expandArticle`.
Here's the fix for the first, where the `eventObj` parameter can be named pretty much anything (even `event`):
```javascript
this.expandButton.addEventListener(
  "click",
  eventObj => this.expandArticle(eventObj)
);
```
and here's the fix to the second, by just not passing an unnecessary argument:
```javascript
this.expandButton.addEventListener("click", () => this.expandArticle());
```